### ISSUE TITLE
Increase code coverage

### ISSUE THREAD

**Issue by ftomassetti** [javaparser/javaparser#820]
We have currently a code coverage of ~60%. We could look into increasing it, perhaps focusing on low hanging fruits. I do not think it makes sense to aim at a very high value but I suppose that with a limited effort we could move the needle.

**Comment by matozoid** [comment]
Shall I tag this "easy"?

**Comment by ftomassetti** [comment]
Yes, I think many tests would be easy to write. Maybe we should just point to where to read the code coverage.

**Comment by matozoid** [comment]
Some thoughts:
- I'm not sure the code generated by JavaCC is included correctly. We never seem to get much above 60%
- a lot of code in src/main/java is generated by us. I don't feel a need to test all of it, so coverage will never be high

**Comment by JulienVig** [comment]
Hi! Is this issue still open?

**Comment by matozoid** [comment]
Yeah, a few more tests are always welcome!

**Comment by AnnaSkantz** [comment]
Hi,
We are a group of students who would like to work on this issue as part of a group assignment.

**Comment by jlerbsc** [comment]
Hi @AnnaSkantz you can submit PR as you wish!

**Comment by ghost** [comment]
Hello, some other students and I would like to work on this for a university project. What would be some good code for us to test and can we get this issue assigned to us? Or should we just submit a PR with our additions without this issue getting assigned?

**Comment by jlerbsc** [comment]
You can indeed propose improvements. Thank you for asking the problem you are encountering or trying to solve.

**Comment by YinyinChencr7** [comment]
Hi, can I work on this issue?

**Comment by jlerbsc** [comment]
Your help will be appreciated.

**Comment by soybeaam** [comment]
Hello!

I was wondering if my team and I could still work on this issue.

**Comment by jlerbsc** [comment]
Hi @soybeaam Thank you for your proposition, but there is no immediate need at this time. First, the coverage ratio should be analysed and the cases with the highest added value identified.
